src/utilities/endpoints/endpoints.ts

```diff
@@ -17,8 +17,8 @@ export type KnownEndpoints = (typeof endpoints)[number];
 
 export const publicEndpoints = {
   BOTLabs: 'wss://spiritnet.kilt.io',
-  Dwellir: 'wss://kilt-rpc.dwellir.com',
-  OnFinality: 'wss://spiritnet.api.onfinality.io/public-ws',
+  IBP1: 'wss://kilt.ibp.network',
+  IBP2: 'wss://kilt.dotters.network',
```

Comment:
Would it make sense to expand the abbreviation "IBP" in this human-readable label? Does it even mean anything?

Reply:
Exanded will be `Infrastructure Builders Programme` ([IBP](https://wiki.ibp.network/docs/basics/introduction)) which I believe is too long for Sporran to display. Moreover it's displayed as IBP1/2 on [Polkadotjs alreads across the board](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.ibp.network%2Fpolkadot#/explorer)